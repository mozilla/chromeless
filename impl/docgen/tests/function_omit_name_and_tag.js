/**
 * A module with a basic function definition in it,
 * which omits all tags, letting the system figure out
 * what we mean.
 */

/**
 * a function which converts URLs to filenames
 */
let toFilename = exports.toFilename = function toFilename(url) {
}
