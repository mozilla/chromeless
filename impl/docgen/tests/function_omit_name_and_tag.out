{
  "desc": "A module with a basic function definition in it,\nwhich omits all tags, letting the system figure out\nwhat we mean.", 
  "filename": "tests/function_omit_name_and_tag.js", 
  "functions": [
    {
      "desc": "a function which converts URLs to filenames", 
      "name": "toFilename"
    }
  ], 
  "module": "function_omit_name_and_tag"
}
